src/Controls/src/Core/Brush/Brush.cs

```diff
@@ -89,442 +92,450 @@ public abstract partial class Brush : Element
 		}
 
 		static ImmutableBrush defaultBrush;
-		/// <include file="../../docs/Microsoft.Maui.Controls/Brush.xml" path="//Member[@MemberName='Default']/Docs/*" />
+
+		/// <summary>
+		/// Represents the default (empty) brush.
+		/// </summary>
 		public static Brush Default => defaultBrush ??= new(null);
 
 		public static implicit operator Brush(Color color) => new SolidColorBrush(color);
 
-		/// <include file="../../docs/Microsoft.Maui.Controls/Brush.xml" path="//Member[@MemberName='IsEmpty']/Docs/*" />
+		/// <summary>
+		/// When overridden in a derived class, indicates whether the given brush represents the empty brush.
+		/// </summary>
 		public abstract bool IsEmpty { get; }
 
-		/// <include file="../../docs/Microsoft.Maui.Controls/Brush.xml" path="//Member[@MemberName='IsNullOrEmpty']/Docs/*" />
+		/// <summary>
+		/// Indicates whether the specified <see cref="Brush"/> is <see langword="null"/> or empty.
+		/// </summary>
+		/// <returns><see langword="true"/> if the brush is null or empty; <see langword="false"/> otherwise.</returns>
 		public static bool IsNullOrEmpty(Brush brush)
 		{
 			return brush == null || brush.IsEmpty;
 		}
 
 		static ImmutableBrush aliceBlue;
-		/// <include file="../../docs/Microsoft.Maui.Controls/Brush.xml" path="//Member[@MemberName='AliceBlue']/Docs/*" />
+		/// <summary>Gets the system-defined color that has an ARGB value of <c>#FFF0F8FF</c>.</summary>
```

Comment:
Should these maybe reference `Colors.AntiqueWhite`, etc here instead of the hex values? Technically that is what they are referring to from here and that also eliminates the need of updating the hex value in 2 files.

Reply:
Sounds good. I'll make the change!